app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,33 @@
+class CommentsController < ApplicationController
+  before_action :set_post
+
+  def create
+    @comment = @post.comments.create(comment_params)
+    redirect_to @post
+  end
+
+  def edit
+    @comment = Comment.find(params[:id])
```

Comment:
We can create a private method called set_comment or something like that to set a @comment in edit, update and destroy